bin/pre-commit-hook.js

```diff
@@ -73,7 +73,7 @@ files.forEach( file => {
 // linting should happen after formatting
 const lintResult = spawnSync(
 	'eslint',
-	[ ...files.filter( file => ! file.endsWith( '.scss' ) ), '--', '--diff=index' ],
+	[ ...files.filter( file => ! file.endsWith( '.scss' ) ) ],
```

Comment:
This is a change I couldn't really get to the bottom of– the added `-- --diff=index` was breaking the eslint command, but it appears to work normally when removed. I'm not clear on what it does in the first place though, it might have been leftover from `eslint-eslines`, which we're not using here.